docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,94 @@ which groups these values were observed, as well as the difference and ratio
 between the maximum and the minimum values. For more information refer to the
 subpackage :code:`fairlearn.metrics`.
 
+Failure modes encountered when abstracting
+--------------------------------------------------
+
```

Comment:
modified to

> What traps can we fall into when abstracting a social problem?

Reply:
I was thinking we can add a sentence after the first one, e.g., something like:

"Machine learning systems used in the real world are inherently sociotechnical
 systems, which include both technologies and social actors. Designers of machine learning systems typically translate a real-world context into a machine learning model through abstraction: focussing only on 'relevant' aspects of that context, which are typically described by inputs, outputs, and the relationship between them. However, by abstracting away the social context they are at risk of falling into 'abstraction traps': a failure to consider how social context and technology is interrelated."

or something like that.

I personally prefer to talk from "we" perspective rather than "they", because it sounds a bit more humble and people might get less "computer science is bad and you should feel bad" vibes. But this is obviously a stylistic choice and I'm happy either way.

WDYT @mmadaio

Reply:
Incorporated and reworded from "they" to "we"